.github/workflows/linux.yaml

```diff
@@ -47,12 +47,17 @@ jobs:
       run: |
         SHORT_SHA=$(echo ${{ env.SHA }} | head -c 8)
         make image
-        docker tag simplyblock/spdkcsi:latest simplyblock/spdkcsi:$TAG
-        docker tag simplyblock/spdkcsi:latest simplyblock/spdkcsi:$TAG-$SHORT_SHA
-        docker push simplyblock/spdkcsi:$TAG
-        docker push simplyblock/spdkcsi:$TAG-$SHORT_SHA
+       
+        docker tag simplyblock/spdkcsi:latest-amd64 simplyblock/spdkcsi:$TAG-amd64
+        docker tag simplyblock/spdkcsi:latest-arm64 simplyblock/spdkcsi:$TAG-arm64
 
+        docker push simplyblock/spdkcsi:$TAG-amd64
+        docker push simplyblock/spdkcsi:$TAG-arm64
+
+        docker buildx imagetools create -t simplyblock/spdkcsi:$TAG simplyblock/spdkcsi:$TAG-amd64 simplyblock/spdkcsi:$TAG-arm64
+
+        docker buildx imagetools create -t simplyblock/spdkcsi:$TAG-$SHORT_SHA simplyblock/spdkcsi:$TAG-amd64 simplyblock/spdkcsi:$TAG-arm64
+    
         if [ "${{ github.ref }}" = "refs/heads/master" ]; then
-          docker tag simplyblock/spdkcsi:$TAG simplyblock/spdkcsi:latest
-          docker push simplyblock/spdkcsi:latest
+          docker buildx imagetools create -t simplyblock/spdkcsi:latest simplyblock/spdkcsi:$TAG-amd64 simplyblock/spdkcsi:$TAG-arm64
```

Comment:
this changes seems to be already present on the master branch. Could you please check why this is happening.